src/components/RequestInvite.js

```diff
@@ -125,12 +125,17 @@ export class RequestInvite extends Component {
     }
     return (
       <div>
-        <b>{<FormattedMessage id="request_invite" values={{
-          full_name: this.props.isToMemberDocLoaded ? this.props.toMemberDoc.get('full_name') : null,
-          mid: this.props.memberId
-        }} />}</b>
+        <b>
```

Comment:
Should use CSS or a proper HTML element instead of a `<b />` element, If this is a header or something I would just use an `<h1234 />` kind of element, or if it's not I would wrap it in a span, give it a class name, and then maybe make a styled component on the parent div like so:

```
import styled from "styled-components";
...
const RequestInviteElem = styled.div`
  > .requesterName {
    font-weight: bold;
  }
`;

return (
  <RequestInviteElem>
    <span className="requesterName"><FormattedMessage ... /></span>
    ...
  </RequestInviteElem>
);
```

Additionally, if this is the main part of the page, you might as well just use a semantic `<main />` element, so change the above code to say

```
const RequestInviteElem = styled.main`
  ...
`;
```

The goal with using semantic elements is to remove as much styling information from the markup as you can, so that the page contents themselves are just a reflection of the underlying content you're presenting, rather than some mixture of content and visual info; it makes it easier to restyle things over time, as well as understand what you're looking at when you inspect the markup of the page, along with other positive externalities.